medkumo.js

```diff
@@ -307,6 +309,16 @@
         return result;
     }
 
+    function concatString(arrayKeys) {
```

Comment:
Pls apply something like this:
function getDoctorAddress() {
  return Config.doctor.city + ", " + Config.doctor.state + ", " + Config.doctor.country
}